addons/l10n_dk_nemhandel/views/res_partner_views.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="res_partner_form_l10n_dk_nemhandel" model="ir.ui.view">
+        <field name="name">res.partner.form.l10n.dk.nemhandel</field>
+            <field name="model">res.partner</field>
+            <field name="inherit_id" ref="account_edi_ubl_cii.view_partner_property_form"/>
+            <field name="arch" type="xml">
+            <data>
+                <xpath expr="//group[@name='electronic_invoicing']" position="inside">
+                    <field name="is_using_nemhandel" invisible="True"/>
+                    <label for="nemhandel_identifier_type" string="Nemhandel Address" invisible="not is_using_nemhandel"/>
+                    <div id="nemhandel_address" invisible="not is_using_nemhandel" class="row">
+                        <field name="nemhandel_identifier_type" class="w-30"/>
+                        <field name="nemhandel_identifier_value" placeholder="Endpoint" class="w-70"/>
+                    </div>
+                    <label for="nemhandel_verification_state"
+                           invisible="not is_using_nemhandel or not nemhandel_identifier_value"/>
+                    <div class="row"
```

Comment:
wondering if all thoses row classes could not be just d-flex but maybe you tried it already

Reply:
d-flex would need other classes/style to avoid them taking all the room. So I would need to introduce paddings, force the repartition of size in another way.
I'm a bit lazy to look. But if you think it could be better, I can